core/arch/arm/plat-rockchip/platform_rk322x.c

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2017, Fuzhou Rockchip Electronics Co., Ltd.
+ * Copyright (C) 2019, Theobroma Systems Design und Consulting GmbH
+ */
+
+#include <common.h>
+#include <io.h>
+#include <mm/core_memprot.h>
+#include <platform.h>
+#include <platform_config.h>
+
+register_phys_mem_pgdir(MEM_AREA_IO_SEC, SGRF_BASE, SGRF_SIZE);
+register_phys_mem_pgdir(MEM_AREA_IO_SEC, DDRSGRF_BASE, DDRSGRF_SIZE);
+
+#define SGRF_SOC_CON(n)		((n) * 4)
+#define DDR_SGRF_DDR_CON(n)	((n) * 4)
+#define DDR_RGN0_NS		BIT32(30)
+#define SLAVE_ALL_NS		0xffff0000
```

Comment:
minor: `GENMASK_32(31, 16)`

Reply:
I've added a separate patch for this to not mix moving code with other changes.